src/pybullet_blocks/envs/cleanup_table_env.py

```diff
@@ -611,7 +624,7 @@ def _setup_wiper(self) -> None:
             mass=scene_description.wiper_mass,
             is_wiper=True,
         )
-        rotation_quat = p.getQuaternionFromEuler([np.pi / 2 + 0.2, 0.5, np.pi])
+        rotation_quat = p.getQuaternionFromEuler([np.pi / 2 - 0.1, 0.5, np.pi + 0.2])
```

Comment:
btw, I added this utility `interactively_visualize_pose` somewhat recently, it's very helpful for identifying poses like this https://github.com/tomsilver/pybullet-helpers/blob/ba14b675656982a1fb135c2339e2c9ac831a9c3d/src/pybullet_helpers/gui.py#L236